pylanetary/utils/core.py

```diff
@@ -204,4 +205,42 @@ def convolve_with_beam(data, beam):
     else:
         psf = beam
     return convolution.convolve_fft(data, psf)
-    
\ No newline at end of file
+
+
+class Planet:
```

Comment:
suggest rename of the class to Body, since we will eventually support moons too

Reply:
Changed.